test/strings/search.jl

```diff
@@ -328,3 +328,7 @@ end
 @test @inferred findall(isequal('a'), "éa") == [3]
 @test @inferred findall(isequal('€'), "€€") == [1, 4]
 @test @inferred isempty(findall(isequal('é'), ""))
+
+# issue #18109
+s_18109 = "fooα🐨βcd3"
```

Comment:
This variable isn't necessary.

Reply:
Oh yeah, I originally had two tests here but realized one was redundant.